docs/getting-started.txt

```diff
@@ -107,64 +150,29 @@ might want to review them to gain an understanding of how the Determined API is
 | ``model_def.py`` | Model definition and training/validation loops.                        |
 +------------------+------------------------------------------------------------------------+
 
-***************************************
- Install and Deploy Determined Locally
-***************************************
-
-To install the Determined library and start a cluster locally, enter:
-
-.. code:: bash
-
-   pip install determined
-   det deploy local cluster-up
-
-If your local machine does not have a supported Nvidia GPU, include the no-gpu option to start a
-cluster:
-
-.. code:: bash
-
-   det deploy local cluster-up --no-gpu
-
-**********************************
- Use a Remote Determined Instance
-**********************************
-
-You can also run these examples on a remote Determined master instance by setting the remote IP
-address before running any of the commands:
-
-.. code:: bash
-
-   export DET_MASTER=<ipAddress>:8080
-
-You can also include the master IP address in the command line:
-
-.. code:: bash
-
-   det -m http://<ipAddress>:8080 experiment create distributed.yaml .
+After gaining basic familiarity with Determined tools and operations, you can experiment in your
+domain by replacing these files with your model data and code, and setting configuration parameters
```

Comment:
After gaining basic familiarity with Determined tools and operations, you can ~experiment in your
domain by replacing these files with your model data and code, and setting configuration parameters~ replaces these files with your own model data, code, and configuration.

Or maybe... what is "in your domain" supposed to tell users?  To me it reads like, "feel free to replace this computer-vision model with a model related to the field of study you are in" which is not something I think we need to bother saying.

Reply:
Done.